src/lib/API/api.js

```diff
@@ -245,20 +245,14 @@ class API {
    * `/verify/facerecognition` post api call
    * @param {Credentials} creds
    */
-  performFaceRecognition(req: FormData): Promise<$AxiosXHR<any>> {
+  performFaceRecognition(req: any): Promise<$AxiosXHR<any>> {
     //return { data: { ok: 1, livenessPassed: true, duplicates: false, zoomEnrollmentId:-1 } } //TODO: // REMOVE!!!!!!!!!!
-    return this.client
-      .post('/verify/facerecognition', req, {
-        headers: {
-          'Content-Type': `multipart/form-data;`,
-        },
-      })
-      .then(r => {
-        if (r.data.onlyInEnv) {
-          return { data: { ok: 1, enrollResult: { alreadyEnrolled: true } } }
-        }
-        return r
-      })
+    return this.client.post('/verify/facerecognition', req).then(r => {
```

Comment:
Why api splitted onto few parts ? 
Probably lib/API/api could contain only core API calls and provide methods for perform custom requests such as 
```
async customGet<R>(url: string): Promise<$AxiosXHR<R>>;
async customPost<T, R>(url: string, data: T): Promise<$AxiosXHR<R>>
```

And feature-related APIs will contain all typings and its methods will just proxy calls to customGet/Post

Reply:
This part is very large, refactoring will require more time and testing. 
@sirpy

Reply:
ok we dont need to refactor at the moment